content/java/concepts/calendar/calendar.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Calendar'
+Description: 'An abstract class that represents dates and time and has methods for converting between a given moment in time and a number of calendar attributes.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Classes'
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Calendar`** [class](https://www.codecademy.com/learn/learn-java-classes-and-methods) is an abstract class that represents dates and time and has methods for converting between a given moment in time and a number of calendar attributes such as `YEAR`, `MONTH`, `DAY_OF_MONTH`, `HOUR`, and so on.
+
+## Syntax
+
+```pseudo
+Calendar calendar = Calendar.getInstance();
+```
+
+## Example
+
+```java
+import java.util.Date;
+import java.util.Calendar;
+
+public class CalendarExample {
+    public static void main(String[] args) {
+
+        Calendar calendar = Calendar.getInstance();
+        int year = calendar.get(Calendar.YEAR);
+        int month = calendar.get(Calendar.MONTH);
+        int date = calendar.get(Calendar.DATE);
+
+        System.out.printf("Today's date is: %d/%d/%d%n", date, month + 1, year);
+    }
+}
+```
+
+The above code will output:
+
+```shell
+Today's date is: 1/8/2023
+```
+
+In the above example, we created a `Calendar` object using the `getInstance()` method. Then we used the `get()` method to get the current year, month, date, and day of the month. Note that the month is zero-indexed, so we added one to it to get the correct month.
```

Comment:
Typically we have a sentence before the example. Please move this before the code block and change the language accordingly. I also think it may be better to use a `set` method to create a specific date. That way the output will always match if someone wants to copy and run the code.